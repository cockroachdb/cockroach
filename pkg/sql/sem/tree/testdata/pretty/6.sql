INSERT INTO kv2(k, v) VALUES ('a', 'b'), ('c', 'd'), ('e', 'f'), ('f', 'g'), (ARRAY[NULL::INT]), (ARRAY[NULL::INT, 1]), (ARRAY[1, NULL::INT]), (ARRAY[NULL::INT, NULL::INT]), (((9 / 3) * (1 / 3))), (2.0), (2.4 + 4.6)
