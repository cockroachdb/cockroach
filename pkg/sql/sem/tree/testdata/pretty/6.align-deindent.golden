// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
INSERT
INTO
	kv2
		(
			k,
			v
		)
VALUES
	(
		'a',
		'b'
	),
	(
		'c',
		'd'
	),
	(
		'e',
		'f'
	),
	(
		'f',
		'g'
	),
	(
		ARRAY[
			NULL::INT
		]
	),
	(
		ARRAY[
			NULL::INT,
			1
		]
	),
	(
		ARRAY[
			1,
			NULL::INT
		]
	),
	(
		ARRAY[
			NULL::INT,
			NULL::INT
		]
	),
	(
		9
		/ 3
		* 1
			/ 3
	),
	(
		2.0
	),
	(
		2.4
		+ 4.6
	)

6:
------
INSERT
  INTO kv2
		(
			k,
			v
		)
VALUES (
		'a',
		'b'
       ),
       (
		'c',
		'd'
       ),
       (
		'e',
		'f'
       ),
       (
		'f',
		'g'
       ),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (
		9
		/ 3
		* 1
			/ 3
       ),
       (
		2.0
       ),
       (
		2.4
		+ 4.6
       )

12:
------------
INSERT
  INTO kv2 (
			k,
			v
           )
VALUES (
		'a',
		'b'
       ),
       (
		'c',
		'd'
       ),
       (
		'e',
		'f'
       ),
       (
		'f',
		'g'
       ),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (
		9
		/ 3
		* 1
			/ 3
       ),
       (
		2.0
       ),
       (
		2.4
		+ 4.6
       )

13:
-------------
INSERT
  INTO kv2 (
			k,
			v
           )
VALUES (
		'a',
		'b'
       ),
       (
		'c',
		'd'
       ),
       (
		'e',
		'f'
       ),
       (
		'f',
		'g'
       ),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (
		9 / 3
		* 1
			/ 3
       ),
       (2.0),
       (
		2.4
		+ 4.6
       )

15:
---------------
INSERT
  INTO kv2 (
			k,
			v
           )
VALUES (
		'a',
		'b'
       ),
       (
		'c',
		'd'
       ),
       (
		'e',
		'f'
       ),
       (
		'f',
		'g'
       ),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (
		9 / 3
		* 1 / 3
       ),
       (2.0),
       (
		2.4
		+ 4.6
       )

17:
-----------------
INSERT
  INTO kv2 (k, v)
VALUES (
		'a',
		'b'
       ),
       (
		'c',
		'd'
       ),
       (
		'e',
		'f'
       ),
       (
		'f',
		'g'
       ),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (
		9 / 3
		* 1 / 3
       ),
       (2.0),
       (
		2.4 + 4.6
       )

18:
------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (
		9 / 3
		* 1 / 3
       ),
       (2.0),
       (2.4 + 4.6)

21:
---------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (
		9 / 3 * 1 / 3
       ),
       (2.0),
       (2.4 + 4.6)

23:
-----------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (
		ARRAY[
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (9 / 3 * 1 / 3),
       (2.0),
       (2.4 + 4.6)

24:
------------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (
		ARRAY[NULL::INT]
       ),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (9 / 3 * 1 / 3),
       (2.0),
       (2.4 + 4.6)

26:
--------------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (ARRAY[NULL::INT]),
       (
		ARRAY[
			NULL::INT,
			1
		]
       ),
       (
		ARRAY[
			1,
			NULL::INT
		]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (9 / 3 * 1 / 3),
       (2.0),
       (2.4 + 4.6)

27:
---------------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (ARRAY[NULL::INT]),
       (
		ARRAY[NULL::INT, 1]
       ),
       (
		ARRAY[1, NULL::INT]
       ),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (9 / 3 * 1 / 3),
       (2.0),
       (2.4 + 4.6)

29:
-----------------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (ARRAY[NULL::INT]),
       (ARRAY[NULL::INT, 1]),
       (ARRAY[1, NULL::INT]),
       (
		ARRAY[
			NULL::INT,
			NULL::INT
		]
       ),
       (9 / 3 * 1 / 3),
       (2.0),
       (2.4 + 4.6)

35:
-----------------------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (ARRAY[NULL::INT]),
       (ARRAY[NULL::INT, 1]),
       (ARRAY[1, NULL::INT]),
       (
		ARRAY[NULL::INT, NULL::INT]
       ),
       (9 / 3 * 1 / 3),
       (2.0),
       (2.4 + 4.6)

37:
-------------------------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'),
       ('c', 'd'),
       ('e', 'f'),
       ('f', 'g'),
       (ARRAY[NULL::INT]),
       (ARRAY[NULL::INT, 1]),
       (ARRAY[1, NULL::INT]),
       (ARRAY[NULL::INT, NULL::INT]),
       (9 / 3 * 1 / 3),
       (2.0),
       (2.4 + 4.6)

187:
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
INSERT
  INTO kv2 (k, v)
VALUES ('a', 'b'), ('c', 'd'), ('e', 'f'), ('f', 'g'), (ARRAY[NULL::INT]), (ARRAY[NULL::INT, 1]), (ARRAY[1, NULL::INT]), (ARRAY[NULL::INT, NULL::INT]), (9 / 3 * 1 / 3), (2.0), (2.4 + 4.6)

210:
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
INSERT INTO kv2 (k, v) VALUES ('a', 'b'), ('c', 'd'), ('e', 'f'), ('f', 'g'), (ARRAY[NULL::INT]), (ARRAY[NULL::INT, 1]), (ARRAY[1, NULL::INT]), (ARRAY[NULL::INT, NULL::INT]), (9 / 3 * 1 / 3), (2.0), (2.4 + 4.6)


